assets/frontend/css/frontend.css

```diff
@@ -56,4 +56,17 @@
     padding: 4px 7px; 
     display: inline-block;
     margin: 3px;
+}
+
+/* box-list styling */
+.taxopress-box-list{
+    display: flex;
+    overflow: scroll;
```

Comment:
Why are you displaying all results with scroll, shouldn't more results go to next line and not scroll?